index.js

```diff
@@ -44,11 +70,23 @@ const evalSol = (commands, options={}) => {
   /** Returns true if the given command is an expression that can return a value. */
   const isExpression = command => (!/[^=]=[^=]/.test(command)) && (!command.startsWith('delete'))
   const lastCommand = commands[commands.length - 1]
+  const commandWithoutSemi = lastCommand.endsWith(';') ? lastCommand.substring(0, lastCommand.length - 1) : lastCommand
 
   let returnType = 'bool'
   const content = commands.slice(0, commands.length - 1).join('\n')
-  const contentWithReturnExpression = commands.join('\n')
-  const sourceFirstPass = template({ content: contentWithReturnExpression, returnType, returnExpression: isExpression(lastCommand) ? lastCommand : 'false;' })
+  let contentWithReturnExpression = commands.join('\n')
+
+  //let sourceFirstPass = template({ content: contentWithReturnExpression, returnType, returnExpression: isExpression(lastCommand) ? lastCommand : 'false;' })
+  const specialCommand = specialGlobals[commandWithoutSemi]
+  const sourceFirstPass = template(specialCommand ? {
+    content,
+    returnType: pluck(specialGlobals[commandWithoutSemi], "returnType").join(", "),
+    returnExpression: "(" + pluck(specialGlobals[commandWithoutSemi], "prop").map(item => commandWithoutSemi + "." + item).join(", ") + ");"
+  }: {
+    content: contentWithReturnExpression,
+    returnType,
```

Comment:
I believe `returnType` in this case will always be `bool`. No need for a local variable when the value can be inlined.